pkg/workloads/cortex/lib/util.py

```diff
@@ -48,9 +51,19 @@ def indent_str(text, indent):
     return indent * " " + text.replace("\n", "\n" + indent * " ")
 
 
+def json_tricks_dump(obj, **kwargs):
```

Comment:
Maybe this should just use `pp_str_flat()` in other files and leave this as a helper internal to `util.py`?